File: no 1/java/product.java

// Latihan 1

// base class: Product
class Product{
	private String price;
	private String id_product;

	// constructor
	Product(){
	}

	public void setPrice(String price){
		this.price = price;
	}
	public String getPrice(){
		return this.price;
	}
	
	public void setId_Product(String id_product){
		this.id_product = id_product;
	}
	public String getId_Product(){
		return this.id_product;
	}

}

File: no 2/python/product.py
class Product:

# kelas yang digunakan untuk mengimplementasikan kelas product

	def __init__(self):
	# konstruktor
		self.price = ""
		self.id_product = ""

	def setPrice(self, price):
	# set nilai atribut price
		self.price = price;

	def getPrice(self):
	# mengembalikan nilai atribut price
		return self.price

	def setId_Product(self, id_product):
	# set nilai atribut id_product
		self.id_product = id_product
	def getId_Product(self):
	# mengembalikan nilai atribut id_product
		return self.id_product